### PR TITLE
A title

### PR DESCRIPTION
**Pull request by CodelingoBotTester**
A body

### CODE REVIEW

File: test.txt [hunk 1]
```diff
@@ -1 +1,4 @@
 Test commit.
+
+More content
```
Comment (CodelingoBotTester) [inline test.txt]:
```diff
- // NoComment is a function
+ // NoComment {{ .commentVar }}
```

Change `commentVar` by commenting: `@CodeLingoBot set commentVar "your comment"`

Comment (CodelingoBotTester) [reply test.txt]: @CodelingoBotTester rewrite set commentVar "your comment"

Comment (CodelingoBotTester) [reply test.txt]: @CodelingoBotTester rewrite set commentVar "your comment"

Comment (Daanikus) [reply test.txt]: @CodelingoBotTester rewrite set commentVar "your comment"

Comment (CodelingoBotTester) [reply test.txt]: Success! Committed rewritten file.

**Pull request closed without merging**